collector/src/main/java/com/bitmovin/analytics/data/DeviceInformationProvider.kt

```diff
@@ -1,14 +1,22 @@
 package com.bitmovin.analytics.data
 
+import android.app.UiModeManager
 import android.content.Context
+import android.content.res.Configuration
 import android.os.Build
 import com.bitmovin.analytics.utils.Util
 
 open class DeviceInformationProvider(val context: Context, val userAgent: String) {
+    private var isTV: Boolean = false
+    init {
```

Comment:
nothing wrong with having this in a function and calling it from `getDeviceInformation` ..